internal/ent/hooks/invite.go

```diff
@@ -74,6 +77,24 @@ func HookInvite() ent.Hook {
 				m.Logger.Errorw("error sending email to user", "error", err)
 			}
 
+			orgID, _ := m.OwnerID()
+			org, _ := m.Client().Organization.Query().Where(organization.ID(orgID)).Only(ctx)
+			reqID, _ := m.RequestorID()
+			requestor, _ := m.Client().User.Query().Where(user.ID(reqID)).Only(ctx)
+			email, _ := m.Recipient()
+			role, _ := m.Role()
```

Comment:
nit: use the `Get` instead of `Query().Where` when you are looking up by ID. 
```suggestion
			org, _ := m.Client().Organization.Get(ctx, orgID)
			reqID, _ := m.RequestorID()
			requestor, _ := m.Client().User.Get(ctx, reqID)
			email, _ := m.Recipient()
			role, _ := m.Role()
```

Reply:
what's the difference?

Reply:
They hit different resolvers ,the `Get` hits the "Get" resolver, and the "Query" his the "List" resolver, just simpler to use if you are looking up by id